config.lua

```diff
@@ -35,7 +39,7 @@ config({'development', 'test'}, {
     site_name = 'dev | Snap Cloud',
     port = os.getenv('PORT') or 8080,
     mail_smtp_port = os.getenv('MAIL_SMTP_PORT') or 1025,
-    dns_resolver = 'localhost',
+    dns_resolver = '8.8.8.8',
```

Comment:
localhost was having issues resolving on the Mac, but using the DigitalOcean IP in production should still be fine.